### DISCUSSION TITLE
[Feature] Store iOS edit metadata including portrait mode details

### DISCUSSION THREAD

**Discussion by gitgizmo7** [ente-io/ente#12354] [Enhancements]
### Description

On iOS, portrait mode photos include extra metadata to allow for edits to the digital blur after the fact such as changing the depth, refocusing, and disabling the effect entirely. When a portrait mode photo is uploaded to Ente, this metadata is lost such that when re-importing the image into the native photos app, the photo is recognized as a portrait photo, but it can no longer be edited as before. It appears that this requires uploading extra files as when such a photo is shared via AirDrop with "all photos data" from iOS to macOS, it results in a folder containing two `.HEIC` files and two `.AAE` files. Reproducing the bug is as simple as backing up a portrait mode photo to Ente, redownloading it (issue affects both downloads from mobile app and desktop/web), importing it into Photos, and observing that compared to the original photo, portrait-specific edits can no longer be made.

### Version

0.8.62

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - iOS

**Comment by gitgizmo7** [comment]
Any update on this? This prevents Ente from being a viable backup solution for me, as I'd like to make sure I don't lose any data. I just re-tested and this is still an issue.

**Comment by AmanRajSinghMourya** [comment]
This is more of a feature request then an issue so moving it to `Discussions`. Will prioritize this on the upvotes
